app/src/main/res/values/strings.xml

```diff
@@ -1104,6 +1104,8 @@ If you are overwhelmed by the number of quests, you can always fine-tune which q
     <string name="quest_fireHydrant_confirmation_title">"Are you sure? Sometimes the diameter can be found on the hydrant itself."</string>
     <string name="short_no_answer_on_button">No</string>
     <string name="quest_level_title">On which level is %s located?</string>
+    <string name="quest_barrier_path_intersection">How does this path go through this barrier?</string>
+    <string name="quest_barrier_road_intersection">How does this road go through this barrier?</string>
```

Comment:
It's hard to say exactly what phrasing is best without seeing the full form. I don't remember why we can't re-use "What type of barrier is this?" (but I do remember it being discussed; I just wasn't paying attention).

In general, I like "pass through" (or maybe "cross") better than "go through".
"Here" is redundant with "this". It should either be "…through this barrier?" or "…through the barrier here?"

Reply:
I took more time to read/think about the context, the reason we can't re-use the other version is because this is for when you have something like a road crossing a fence or rock wall. So, I think this is my favorite so far:

```suggestion
    <string name="quest_barrier_path_intersection">How does this path cross the barrier here?</string>
    <string name="quest_barrier_road_intersection">How does this road cross the barrier here?</string>
```

I still don't really like the term *barrier* here. It's probably technically correct, but it makes me thing of something which is constructed specifically to **block** the road, whereas in this case the wall is blocking a big section, and the gate is constructed specifically to allow the road through. I'm not sure what the right word is, though.

Reply:
Maybe something like "What allows this road through the barrier here?" @peternewman any ideas?

Reply:
> I don't remember why we can't re-use "What type of barrier is this?" 

see https://github.com/streetcomplete/StreetComplete/pull/3515#issuecomment-996073809 for my reasons (though @westnordost  also was not a fan so maybe I should drop this idea)

Reply:
Alright, then "cross the barrier here" is best because it clearly refers to the crossing (which is highlighted), not ot the road and way

Reply:
Sorry for being late to the party, I'd favour @smichel17 's "allows this road through the barrier" over "cross the barrier"; a bridge crosses a river, but a road/path doesn't cross a wall it goes through it (or they intersect, but that sounds a bit clunky).

Pedantically I'd argue the wall is bisected/cut by the road, i.e. technically there is no wall at the road, but something else (what we're trying to find out) at that point.